bucket/opencv.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "4.4.0",
+    "url": "https://sourceforge.net/projects/opencvlibrary/files/4.4.0/opencv-4.4.0-vc14_vc15.exe/download#/dl.7z",
+    "hash": "cd480a6fe345f4285bf01626998ef8d65b92fbfb2db9f2905034e4c4786e0d50",
+    "homepage": "https://opencv.org",
+    "extract_dir": "opencv",
+    "checkver": {
+        "url": "https://opencv.org/releases/",
+        "regex": "opencv-([\\d\\.]+)-vc.+?\\.exe"
+    },
+    "autoupdate": {
+        "url": "https://sourceforge.net/projects/opencvlibrary/files/$version/opencv-$version-vc14_vc15.exe/download#/dl.7z",
+        "extract_dir": "opencv"
```

Comment:
```suggestion
    "version": "4.4.0",
    "desciption": "Computer vision and machine learning software library",
    "homepage": "https://opencv.org",
    "license": "BSD-3.0-clause",
    "architecture": {
        "64bit": {
            "url": "https://downloads.sourceforge.net/project/opencvlibrary/4.4.0/opencv-4.4.0-vc14_vc15.exe#/dl.7z",
            "hash": "sha1:f6a22ce6e407d307da0d839b8553ec8d4ad40959",
            "bin": [
                "x64\\vc15\\bin\\opencv_annotation.exe",
                "x64\\vc15\\bin\\opencv_interactive-calibration.exe",
                "x64\\vc15\\bin\\opencv_visualisation.exe"
            ]
        }
    },
    "extract_dir": "opencv\\build",
    "checkver": {
        "url": "https://opencv.org/releases/",
        "regex": "OpenCV\\s+(?:-|&#8211;)\\s+([\\d.]+)"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://downloads.sourceforge.net/project/opencvlibrary/$version/opencv-$version-vc14_vc15.exe#/dl.7z"
            }
        }
```

Reply:
I'd keep the source directory if someone wants to build samples there

Reply:
No. Building is out of scope of scoop and if someone want to build it, then build it according to contributing.